libfwupdplugin/fu-bios-attrs.c

```diff
@@ -155,6 +155,25 @@ fu_bios_attr_set_current_value(FwupdBiosAttr *attr, GError **error)
 #define LENOVO_READ_ONLY_NEEDLE "[Status:ShowOnly]"
 #define LENOVO_EXCLUDED		"[Excluded from boot order:"
 
+static void
+fu_bios_attr_fixup_read_only(FwupdBiosAttr *attr)
+{
+	if (fwupd_bios_attr_get_kind(attr) == FWUPD_BIOS_ATTR_KIND_ENUMERATION) {
+		const gchar *id = fwupd_bios_attr_get_id(attr);
+		const gchar *tmp = fwupd_bios_attr_get_current_value(attr);
+		if (g_strcmp0(id, "com.thinklmi.SecureBoot") == 0 &&
+		    g_strcmp0(tmp, "Enable") == 0) {
```

Comment:
I honestly think we need a struct here that's being looped; there's going to be quite a few of these I suspect.

Reply:
Can you explain better?  I don't follow.

Reply:
I thought of something like https://github.com/fwupd/fwupd/blob/main/src/fu-udev-backend.c#L31